README.md

```diff
@@ -337,11 +337,8 @@ If you fork the project on GitHub, you can run your fork's [build workflow](.git
                                     configuration files
     --plugin-dirs PATH              Path to an additional directory to search
                                     for plugins. This option can be used
-                                    multiple times to add multiple directories.
-                                    Note that this currently only works for
-                                    extractor plugins; postprocessor plugins can
-                                    only be loaded from the default plugin
-                                    directories
+                                    multiple times to add multiple directories
+    --no-plugins                    Disable plugin loading
```

Comment:
Do we want to convey that `--plugin-dirs /path --no-plugins --plugin-dirs /new/path` will work? any suggestions?

Reply:
`Clear plugin directories to search, including defaults and those provided by previous --plugin-dirs` perhaps?

Reply:
https://github.com/yt-dlp/yt-dlp/pull/11305/commits/b6c3d22decc56b8484de0435aa52948d6a59b217